### PR TITLE
Structure logging via Serilog

### LINKED ISSUE #5: Structured logging
I want the logs to be stored in some structured storage (e.g. check how ELMAH does its deeds). Provide log storage and a way to read the logs (and to maintain the storage if needed).

### PR DESCRIPTION
**Pull request by gsomix**
First steps toward #5 

- [x] Console logging
- [x] Replace Literate sink with Console

### CODE REVIEW

File: Nightwatch/Nightwatch.fsproj [hunk 1]
```diff
@@ -19,6 +19,8 @@
     <PackageReference Include="Argu" Version="3.7.0" />
     <PackageReference Include="FSharp.Control.FusionTasks.FS41" Version="1.1.0" />
     <PackageReference Include="Quartz" Version="3.0.0-alpha3" />
+    <PackageReference Include="Serilog" Version="2.5.0" />
+    <PackageReference Include="Serilog.Sinks.Literate" Version="3.0.0" />
```
Comment (ForNeVeR) [inline Nightwatch/Nightwatch.fsproj]: [This package is being retired](https://github.com/serilog/serilog-sinks-literate#this-package-is-being-retired), so let's switch to console sink?

**Comment by ForNeVeR** [comment]
Maybe let's add additional logs to the shell and HTTP runners then?

**Comment by ForNeVeR** [comment]
Possible issue of interest: https://github.com/dotnet/corefx/issues/23592

(Seems like .NET Core going to add an API to work with a process argument list.)

**Comment by gsomix** [comment]
OK, I have updated PR description. Let's do this small step!

**Comment by ForNeVeR** [comment]
Thanks for the contribution!

**Pull request merged**